pkg/provision/storage/cleanup.go

```diff
@@ -34,6 +28,12 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	"github.com/devfile/devworkspace-operator/internal/images"
+	"github.com/devfile/devworkspace-operator/pkg/common"
+	"github.com/devfile/devworkspace-operator/pkg/config"
+	"github.com/devfile/devworkspace-operator/pkg/constants"
+	workspace2 "github.com/devfile/devworkspace-operator/pkg/provision/workspace"
```

Comment:
IDE's footprint is found

Reply:
I just wish it would warn me it was doing this :smile: 

Thanks for catching it.